README.md

```diff
@@ -92,6 +93,24 @@ The above output includes only software that is visible in the container (i.e.,
 syft <image> --scope all-layers
 ```
 
+#### Format conversion - __experimental__
+
+Syft can convert between different SBOM formats, such as CycloneDX, SPDX, and Syft's own format. The ability to convert SBOMs helps saving the time and effort it would take to recreate SBOMs from container images, or file systems.
+
+``` bash
+syft convert sbom.syft.json -o cyclonedx-json=sbom.cdx.json
+```
+
+This feature is experimental and data might be lost when converting formats. Packages are the main SBOM component easily transferable across formats, while files and relationships are more likely to lose data. 
+
+The supported formats are:
```

Comment:
We may have covered this, but refresh me: do we not want to support all formats?

Reply:
We support formats with wide community usage AND encode/decode support. The recent GitHub format is niche and doesn't have a decoder, therefore was excluded.

The exception to the rule above is table format, which is the default output for two reasons:
1. it matches syft's default SBOM output
2. it is handy, as a summary, to print SBOMs as tables

Reply:
@luhring I will add the explanation above to the README so others can understand the design decision.

Reply:
I appreciate that, @jonasagx ! I actually think it's okay to leave out the explanation from the README, I just wanted to understand our strategy for myself.

I don't think we need to exclude the GH format necessarily, but I'm okay deferring this. If the error messages are good, like "destination format %q does not have an encoder" or something, I think users will know what to ask for if they want expanded support.